data_rentgen/server/services/lineage.py

```diff
@@ -870,3 +929,52 @@ async def _dataset_lineage_with_job_granularity(
             result.jobs = {job.id: job for job in jobs}
 
         return result
+
+    async def _get_column_lineage(
+        self,
+        current_result: LineageServiceResult,
+        since: datetime,
+        until: datetime | None,
+        granularity: Literal["OPERATION", "RUN", "JOB"],
+    ) -> LineageServiceResult:
+        """
+        'granularity' argument of this function not the same as granularity in api request.
+        Here it's used for aggregation entity
+        """
+        result = LineageServiceResult()
+        result.column_lineage = defaultdict(list)
```

Comment:
If we return only column_lineage, could we just return `list[ColumnLineageRow]` instead of `LineageServiceResult` here?